application/contract/rootdomain/rootdomain.go

```diff
@@ -97,12 +97,15 @@ func (rd *RootDomain) AddBurnAddress(burnAddress string) error {
 	return nil
 }
 
-func (rd RootDomain) GetBurnAddress() (string, error) {
+func (rd *RootDomain) GetBurnAddress() (result string, err error) {
```

Comment:
Functions and methods should not use named return parameters (our code style, 3.2)

Reply:
In this case it is not get method in standard meaning.

Reply:
RootDomain doesn't have field burnAddress. Instead in this method we get one burn address from list.

Reply:
I'm talking about return parameters, not get prefix in method name